requirements.txt

```diff
@@ -1,4 +1,4 @@
 cython>=0.29.20
 numpy>=1.16.6
-scipy>=1.0
```

Comment:
I discussed this with @Ericgig and remembered that for QuTiP v5 we are planning to support scipy 1.5+. Could you update the version here to 1.5 and then it's done?

Reply:
@hodgestar Done